src/main/java/seedu/address/ui/CommandBox.java

```diff
@@ -17,18 +23,54 @@
     private static final String FXML = "CommandBox.fxml";
 
     private final CommandExecutor commandExecutor;
+    private final Queue<String> commandQueue;
 
     @FXML
+    private StackPane stackPane;
     private TextField commandTextField;
 
     /**
      * Creates a {@code CommandBox} with the given {@code CommandExecutor}.
+     * When multiple lines of commands are pasted into the box,
+     * pressing enter will execute the first command and autofill the next one.
+     * This continues until a different command is entered or all commands have been entered.
      */
     public CommandBox(CommandExecutor commandExecutor) {
         super(FXML);
         this.commandExecutor = commandExecutor;
+        this.commandQueue = new LinkedList<>();
+        initCommandBox();
```

Comment:
Can the SLAP be improved here?
One way is to rename initCommandBox() to initTextField()
and move ```stackPane.getChildren().add(commandTextField);``` to constructor of ```CommandBox```